docs/metrics/dimensions.yaml

```diff
@@ -29,6 +29,19 @@ dest.workload.name:
   brief: See workload.name
   description: See workload.name
 
+workload.uid:
+  brief: Kubernetes workload or application UUID.
+  description: If the associated metric comes from a monitored Kubernetes cluster running the k8-collector, the metric may note the UUID of the source and destination workload that the traffic being measured moved between.  
```

Comment:
is it correct that the UID is the owner's, i.e., it would be the DaemonSet or Deployment UID rather than the Pod's UID, i.e., it should not increase the cardinality of data?

If so could you add something that explains it (that it is UID of DaemonSet/Deployement etc) here in the doc, and then this patch looks good to me. Thank you for the contribution!

Reply:
Great, thank you!